custom_model_runner/datarobot_drum/resource/components/Python/prediction_server/prediction_server.py

```diff
@@ -91,6 +91,20 @@ def predict():
                 self._stats_collector.disable()
             return response, response_status
 
+        @model_api.route("/transform/", methods=["POST"])
```

Comment:
You also need to add new endpoint into `uwsgi_component`

Reply:
thanks! what's the right place to add test cases for that one?

Reply:
test_custom_models_with_drum_nginx_prediction_server
notice, these tests run inside docker container to make sure we install nginx properly.  And this container only has deps to run sklearn.
Ping me in slack if need help.

Reply:
thanks - I think I got it up, but having trouble running the test locally (presumably for docker image reasons)